p2p/libp2p/networksharding/listNoKadSharder.go

```diff
@@ -0,0 +1,69 @@
+package networksharding
```

Comment:
package name networkSharding - the same with name of the folder

Reply:
we will stop using camelCase in package naming